### DISCUSSION TITLE
Tie a tie tutorial using model-viewer

### DISCUSSION THREAD

**Discussion by filahf** [google/model-viewer#1961] [Show and tell]
I made a necktie tutorial using model-viewer. The project is part of my master's thesis that I'm currently writing. 
I would love to hear your thoughts!

https://tieatiear.com

 [Video preview](https://streamable.com/mf17i2)

**Comment by elalish** [comment]
Very nice animations and annotations! https://twitter.com/modelviewer/status/1353793911894884353

**Reply by filahf** [reply]
Thank you, I'm glad you liked it!

**Reply by pixlbndr** [reply]
@filahf Awesome project! Would you mind sharing your code? I am looking to do something like this but in a desktop browser, non-AR.

**Reply by filahf** [reply]
Hey @pixlbndr, thank you! Sure I can publish the code, I'll ping you once it's public.

**Comment by Cebolitoss** [comment]
Hey man, I'm trying to make something similar, but in my approach I needed to use two animations on my model, and play it and stop at specific point of the animation. And if I change to the other animation, it should stop at specific point too. My animations are NLA (no-linear).

I'm assuming your animation is Linear, is it correct? How do you stop the animation at the specific points?
How could I do that using my NLA animations? Are you able to point me in the correct direction ?
